src/control/controllers/PidTwoAxesController.cpp

```diff
@@ -0,0 +1,36 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#include <control/controllers/PidTwoAxesController.h>
+
+/**
+ * Create a PID controller for two axes, with different parameters
+ * @param pidXParams a tuple with 4 parameters for X dir PID - proportional, integrative, derivative and feedforward gain
+ * @param pidYParams a tuple with 4 parameters for Y dir PID - proportional, integrative, derivative and feedforward gain
+ */
+PidTwoAxesController::PidTwoAxesController(std::tuple<double, double, double, double> pidXParams, std::tuple<double, double, double, double> pidYParams){
+    pidXAxis = PidController(pidXParams);
+    pidYAxis = PidController(pidYParams);
+}
+
+/**
+ * Computes the PID output for the two axes
+ * @param setPointInput the setpoint expressed as a Vector2
```

Comment:
This param documentation does not give me any more information. I indeed know that it is a setpoint, because the variable is called a setPoint and I know that it is a Vector2, because the type of this variable is Vector2